app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :comments
+
+  def self.update_post_counter(param)
+    @user = User.find_by_name(param[:current_user])
+    @user.update(postcounter: param[:updateTo])
+  end
```

Comment:
- [ ] It would be nice if you use one of the [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `update_post_counter` method every time a new post has been saved to a database, then make the `update_post_counter` method private